crypto/cipher/aes256cbc/decrypter.go

```diff
@@ -40,7 +39,7 @@ type Decrypter struct {
 func (d Decrypter) Decrypt(data mc.EncryptedContent) (mc.PlainContent, error) {
 	encryptedData, err := bytesDecode(data)
 	if err != nil {
-		return nil, errors.WithMessage(err, "could not convert encryptedData")
+		return nil, mc.ErrDecrypt
 	}
 
 	return decryptEncryptedData(d.privateKey, encryptedData)
```

Comment:
I missed this one, but `decryptEncryptedData(d.privateKey, encryptedData)` also needs to return `mc.ErrDecrypt` as it directly returns an error.